feat: key value store backed by filesystem (#20577)
microsoft/playwright: docs/src/test-api/class-teststore.md

# class: TestStore
* since: v1.31
* langs: js

> Playwright Test provides a global `store` object for passing values between project test and tests. It is

Reviewer: ```suggestion
Playwright Test provides a global `store` object that can be used in global setup and tests.
```

> an error to call store methods outside of test and tests.

Reviewer: ```suggestion
```

> 

Reviewer: We'll need more docs :)

```js tab=js-js
const { test, store } = require('@playwright/test');

> test('sign in', async ({ page, context }) => {

Reviewer: Let's do a beforeAll example with a lazy login instead?

  // Perform sign in steps.
  // Save signed-in state to an entry named 'test-user'.
  const contextState = await context.storageState();
  await store.set('test-user', contextState)
});
```

```js tab=js-ts
import { test, store } from '@playwright/test';

test('sign in', async ({ page, context }) => {
  // Perform sign in steps.
  // Save signed-in state to an entry named 'test-user'.
  const contextState = await context.storageState();
  await store.set('test-user', contextState)
});
```

## async method: TestStore.get
* since: v1.31
- returns: <[any]>

Get named item from the store. Returns undefined if there is no value with given name.

### param: TestStore.get.name
* since: v1.31

> - `name` <[string]>

Reviewer: Let's call it `path`?
